src/studio/src/designer/frontend/ux-editor/components/config/SelectionEditComponent.tsx

```diff
@@ -216,16 +224,17 @@ export class SelectionEditComponent
                     >
                       <Grid item={true} classes={{ item: this.props.classes.gridItem }} >
                         <Typography classes={{ root: this.props.classes.textBold }}>
-                          {`${(this.props.type === 'radiobuttons') ?
+                          {`${(this.props.type === 'RadioButtons') ?
                             this.props.language.ux_editor.modal_radio_button_increment :
                             this.props.language.ux_editor.modal_check_box_increment} ${index + 1}`}
                         </Typography>
                       </Grid>
                       <Grid item={true} classes={{ item: this.props.classes.gridItem }}>
                         {renderSelectTextFromResources('modal_text',
-                          this.props.handleUpdateOptionLabel.bind(this, index),
+                          updateLabel,
                           this.props.textResources,
                           this.props.language,
+                          option.label, // Check if the component works as intended
```

Comment:
Can this comment be removed?

Reply:
The comment is there as I didn't really understand the actual component. It seems to work the same way as before, but I may need walkthrough. The actual component didn't make too much sense to me, maybe a candidate for a re-write? (I originally had a TODO in that comment, but Sonar doesn't accept new TODO's :))